designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -90,7 +91,9 @@ object ProcessRepository {
       sourceEnvironment: String,
       targetEnvironment: String,
       sourceScenarioVersionId: Option[VersionId],
-  ) extends ModifyProcessAction
+  ) extends ModifyProcessAction {
+    override val comment: Option[Comment] = None
```

Comment:
doesn't it mean that the `comment: Comment` field should be defined in one case and not be defined at the sealed trait level?

Reply:
done